export_environment_preqrequisites.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
```

Comment:
how is this meant to be called?

trying

```
./export_environment_preqrequisites.sh
```
and then
```
echo $REGION_0
```
but it's empty

Reply:
guess it should be 

```
source ./export_environment_preqrequisites.sh
```
?

Reply:
nvm, saw the answer in https://github.com/camunda/camunda-docs/pull/3430 with `. <script-name>`